frontend/src/pages/Home.jsx

```diff
@@ -160,21 +161,26 @@ const Home = () => {
   });
 
   // state management for which layout to display
-  const [docLayout, setDocLayout] = useState(HOMEPAGE_DOC_LAYOUTS.grid);
+  const [docLayout, setDocLayout] = useState(HOMEPAGE_DOC_LAYOUTS.grid)
+  const [docSection, updateDocSection] = useState(LocalStore.DOC_STATE.New)
+
+  var docsInSection = docState.docs.filter(doc => {
+    return doc.docState == docSection
```

Comment:
`===`, not `==`

nit but you don't really need the `return` here, you can go
`.filter(doc => doc.docState === docSection)`